<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Telegram SpamBot</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow{
	background-color: qlineargradient(spread:pad, x1:0.534, y1:0.528682, x2:1, 		y2:0, stop:0 rgba(0, 104, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
   </property>
   <widget class="QTableWidget" name="users_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>300</width>
      <height>551</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>551</height>
     </size>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <column>
     <property name="text">
      <string>user_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>checkmark</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_users_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get users from :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="group_line">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string extracomment="Uga"/>
    </property>
   </widget>
   <widget class="QPushButton" name="delay_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay (sec)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delay_line">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="console">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="console_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color : rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Sent/All</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sent_all">
       <property name="styleSheet">
        <string notr="true">color : rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset checkmarks to False</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-8</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image : url(:/newPrefix/photo_2021-06-29_11-18-24.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/newPrefix/photo_2021-06-29_11-18-24.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>users_table</zorder>
   <zorder>label</zorder>
   <zorder>message_line</zorder>
   <zorder>message_label</zorder>
   <zorder>message_button</zorder>
   <zorder>get_users_button</zorder>
   <zorder>group_line</zorder>
   <zorder>delay_button</zorder>
   <zorder>delay_line</zorder>
   <zorder>console</zorder>
   <zorder>console_label</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>reset_button</zorder>
   <zorder>quit_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
